paddle/phi/api/yaml/ops.yaml

```diff
@@ -1786,6 +1786,16 @@
     func : nonzero
     data_type: condition
 
+- op : norm_helper
+  args : (Tensor x, Tensor residual, Tensor bias, Tensor norm_weight, Tensor norm_bias, float epsilon, float residual_alpha, str norm_type, int begin_norm_axis)
```

Comment:
I think more descriptions about the `norm_type`, `begin_norm_axis`, `residual_alpha`  should be added.